index.js

```diff
@@ -170,6 +175,30 @@ class PostHog {
         return this
     }
 
+    /**
+     * @description Sets a groups properties, which allows asking questions like "Who are the most active companies"
+     * using my product in PostHog.
+     *
+     * @param groupType Type of group (ex: 'company'). Limited to 5 per project
+     * @param groupKey Unique identifier for that type of group (ex: 'id:5')
+     * @param properties OPTIONAL | which can be a object with any information you'd like to add
+     */
+    groupIdentify(message, callback) {
+        this._validate(message, 'groupIdentify')
+
+        const captureMessage = {
+            event: '$groupidentify',
+            distinctId: `\$${message.groupType}_${message.groupKey}`,
```

Comment:
Seems different from the docs: https://posthog.com/docs/user-guides/group-analytics#updating-group-properties (distinctId inside properties vs outside)

Reply:
Good question. This is intentional - `.capture` expects distinctId at top level and then munges it into the right spot in the actual API message.

https://github.com/PostHog/posthog-node/blob/master/index.d.ts#L24-L33